Lesson9/anna/Lesson9-aysel/styles.css

```diff
@@ -0,0 +1,214 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  display: grid;
```

Comment:
I'd not use grid for the article element. The text content is too large and in reality it also can vary quite a lot. Grid is better suited for multidirectional layouts. 
In fact, I'd not make the article flex or grip container at all. 
However, I'd all the content that is about the "clock-image" in a div and make that div either grid or flex container to alight the items there accordingly.